Multisig/Logic/Models/Address.swift

```diff
@@ -68,6 +68,10 @@ struct Address: Hashable, ExpressibleByStringInterpolation, CustomStringConverti
         self == .zero
     }
 
+    var isEmpty: Bool {
```

Comment:
In my opinion, this should not be part of the Address type, but it should be a function in the GnosisSafe class, because this is a business logic that belongs there.

Please move it to the GnosisSafe class

Reply:
@DmitryBespalov I think it should be part of address itself actually, I'm thinking about it as attribute of the address that can be used by other business logic parts, same as isZero, the current case is that it's used in the UI , what do you think?